src/factory/FundFactory.sol

```diff
@@ -172,7 +171,7 @@ contract FundFactory is AmguConsumer, Factory {
     {
         managersToRoutes[msg.sender].participation = participationFactory.createInstance(
             managersToHubs[msg.sender],
-            managersToSettings[msg.sender].defaultAssets,
+            managersToSettings[msg.sender]._defaultInvestmentAssets,
```

Comment:
is this being accessed properly (should drop the `_` I think)? @luongnt95

Reply:
yes and fixed.